.changeset/nasty-seas-marry.md

```diff
@@ -0,0 +1,9 @@
+---
+'@leafygreen-ui/code': major
+'@leafygreen-ui/syntax': major
+---
+
+- Adds the line highlighting feature! This feature is supported through the `highlightLines` prop.
```

Comment:
nit: prefer to keep these separate and document the changes specific to each package in their own changeset

Reply:
I can document the multiline change separately, but all the others apply to both components.

Reply:
Wfm